benchmark.py

```
@@ -0,0 +1,318 @@
+    for i in range(N_gc):
+        geodesic_vincenty(ca[i], cb[i])
+
+t_gc, _ = time_func(run_gc, repeats=3)
+t_vin, _ = time_func(run_vin, repeats=3)
+
+print(f"\n  {N_gc:,} sequential calls:")
+print(f"  {'Great Circle (Numba)':<30} {fmt_time(t_gc):>12}")
+print(f"  {'Vincenty Geodesic (Numba)':<30} {fmt_time(t_vin):>12}")
+print(f"  Great Circle is {t_vin/t_gc:.1f}x faster than Vincenty")
+print()
+print(f"  Accuracy trade-off:")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)